server/src/main/java/org/opensearch/wlm/stats/QueryGroupStats.java

```diff
@@ -155,6 +164,9 @@ public static void writeTo(StreamOutput out, QueryGroupStatsHolder statsHolder)
             out.writeVLong(statsHolder.rejections);
             out.writeVLong(statsHolder.failures);
             out.writeVLong(statsHolder.totalCancellations);
+            if (out.getVersion().onOrAfter(Version.V_2_18_0)) {
+                out.writeVLong(statsHolder.shardCompletions);
+            }
```

Comment:
Are these stats changes intentional? Or the branch needs to be rebased?

Reply:
I guess this is not needed.